kubehub/models/CloudProvider.py

```diff
@@ -1,5 +1,6 @@
 from django.db import models
 
+
 class CloudProvider(models.Model):
     CP_TYPES = (
         ('PROX_MOX', 'PROX_MOX'),
```

Comment:
why 2 similar one?

Reply:
first is stored in BD
second used for frontend rendering